src/main/java/com/sda/eventapp/EventAggregationAppApplication.java

```diff
@@ -1,13 +1,32 @@
 package com.sda.eventapp;
 
+import com.sda.eventapp.model.User;
+import com.sda.eventapp.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.security.crypto.password.PasswordEncoder;
 
+@RequiredArgsConstructor
```

Comment:
I'm getting a feeling that this shouldn't be in this class. This is just to start Spring boot, but if I am mistaken, please inform me.

Reply:
Honestly, I do not know. I need this Annotation to create constructor user repository and passwordencoder. But i can changed place and i can create admin in UserService. For example User Service implements CommandLineRunner ...
 What do you think?